bin/verify-exercises

```diff
@@ -21,15 +21,19 @@
 # Verify the Concept Exercises
 for concept_exercise_dir in ./exercises/concept/*/; do
     if [ -d $concept_exercise_dir ]; then
-        echo "Checking $(basename "${concept_exercise_dir}") exercise..."
-        # TODO: run command to verify that the exemplar solution passes the tests
+        slug=$(basename "${practice_exercise_dir}"
+        echo "Checking ${slug} exercise..."
+        cp "${practice_exercise_dir}"/.meta/exemplar.sql "${practice_exercise_dir}/${slug}".sql
+        bin/run.sh "${slug}" "${practice_exercise_dir}" "${practice_exercise_dir}"
```

Comment:
```suggestion
        cp "${concept_exercise_dir}"/.meta/exemplar.sql "${concept_exercise_dir}/${slug}".sql
        bin/run.sh "${slug}" "${concept_exercise_dir}" "${concept_exercise_dir}"
```

Reply:
I have more thoughts about what a verify script should be able to do. But I'll hold those until after this PR is merged.